packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m

```diff
@@ -39,6 +41,31 @@ - (dispatch_queue_t)methodQueue
     }];
 }
 
+RCT_EXPORT_METHOD(getIdentities:(RCTPromiseResolveBlock) resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    
+    [AEPMobileEdgeIdentity getIdentities:^(AEPIdentityMap * _Nullable IdentityMap, NSError * _Nullable error) {
+        
+        if (error) {
+            [self handleError:error rejecter:reject errorLocation:@"getIdentities"];
+        } else {
+            resolve([RCTAEPEdgeIdentityDataBridge dictionaryFromIdentityMap:IdentityMap]);
+        }
+    }];
+}
+
+RCT_EXPORT_METHOD(updateIdentities:(nullable NSDictionary*) map) {
+    AEPIdentityMap *convertMap = [RCTAEPEdgeIdentityDataBridge dictionaryToIdentityMap:map];
+
+    [AEPMobileEdgeIdentity updateIdentities:(AEPIdentityMap * _Nonnull) convertMap];
+}
+
+RCT_EXPORT_METHOD(removeIdentity:(nullable NSDictionary*)item
+                  namespace:(NSString *)namespace) {
+    
+    AEPIdentityItem *convertItem = [RCTAEPEdgeIdentityDataBridge dictionaryToIdentityItem:item];
+    [AEPMobileEdgeIdentity removeIdentityItem:(AEPIdentityItem * _Nonnull) convertItem withNamespace:(NSString * _Nonnull) namespace];
```

Comment:
same comment here, item is nullable, but then the AEPMobileEdgeIdentity removeIdentityItem expects a nonnull, make sure the items is not null before calling this api

Reply:
What to do if the dictionary is nil in dictionaryToIdneityItem?  I return  it to nil right now.
if (!dict) {
        return nil;
    }

Reply:
First of all dictionaryToIdentityItem should not be null but if it evaluates to null you send null, and you cannot pass null item to removeIdentityMap, so you can just log an error and exit

Reply:
Can we have something like 
```
if (!item || !namespace) { 
return; 
}
[AEPMobileEdgeIdentity removeIdentityItem:convertItem withNamespace: namespace];
```